code/controllers/subsystem/mapping.dm

```diff
@@ -257,7 +257,10 @@ Used by the AI doomsday and the self-destruct nuke.
 	// load the maps
 	for (var/P in parsed_maps)
 		var/datum/parsed_map/pm = P
-		if (!pm.load(1, 1, start_z + parsed_maps[P], no_changeturf = TRUE))
+		var/bounds = pm.bounds
+		var/x_offset = bounds ? round(255 / 2 - bounds[MAP_MAXX] / 2) : 0
```

Comment:
dont use hardcoded 255, use maxx and maxy

Reply:
I didn't find the define. Can you please point at it?

Reply:
`world.maxx` and `world.maxy`